dev-requirements.txt

```diff
@@ -1,15 +1,12 @@
-# Additional pip requirements file for pywbem develop.
-#
-# The order of packages is significant, because pip processes them in the order
-# of appearance.
-#
-# Make sure that the minimum versions required in this file are consistent with
-# the minimum versions specified in minimum-constraints.txt.
+# Pip requirements file for development dependencies.
 
 
+# Include test dependencies (which includes install dependencies)
 -r test-requirements.txt
 
-# Direct dependencies:
+
+# Direct dependencies for development and indirect dependencies for development
```

Comment:
Pull these into one line for readability

Reply:
That makes the line longer than my terminal width.